e2e/src/step_definitions/media-viewer.ts

```diff
@@ -122,11 +128,29 @@ const addComment = async (comment: string) => {
   await page.clickOnSaveButton();
 };
 
```

Comment:
I think we should leave this file just for mapping cucumber expressions (Given, When, Then) and move the other functions to another file, unless that breaks the test

Reply:
This is an pre-existing function in the file and has not been changed - will need to be refactored out at some other point in the future.